azerothcore/azerothcore-wotlk
# fix(DB/Gameobject): add chest pooling to battered chests in Jangolade…

… Mine



## Changes Proposed:
-  Limits 1 chest being spawned at any time by adding pooling to the chest
-  Set respawn time to six min on each. (Was on two of them before)

## Issues Addressed:

- Closes #6458

## SOURCE:


## Tests Performed:

- Performed the described how to test the changes.


## How to Test the Changes:


1. Apply SQL
2. There should be no deleted pool templates
3. Check that the pool templates are added correctly 
    - There should be a pool template with entry 11670
    - It should have a limit of 1
    - Entry 11670 should have a description mentioning Jangolade Mine
4. There should be no deleted pool gameobjects
5. Each pool should contain the GUIDs as specified in step 3 and proposed changes, each with a distinct description matching their corresponding zone.
6. These GUIDs should be in no other pools.
7. Check in game for each zone that only one chest has spawned

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
